masterlist.yaml

```diff
@@ -1004,6 +1004,18 @@ plugins:
     url: [ 'https://www.nexusmods.com/newvegas/mods/62041/' ]
     group: *fixesGroup
 
+  - name: 'NVSE/Plugins/sr_New_Vegas_Stutter_Remover.dll'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/34832/'
+        name: 'New Vegas Stutter Remover'
+    msg:
+      - <<: *useInstead
+        subs: [ '[NVTF - New Vegas Tick Fix](https://www.nexusmods.com/newvegas/mods/66537/)' ]
+      - <<: *useInstead
+        subs: [ '[New Vegas Heap Replacer](https://www.nexusmods.com/newvegas/mods/69779/)' ]
+      - <<: *useInstead
+        subs: [ '[lStewieAl''s Tweaks and Engine Fixes](https://www.nexusmods.com/newvegas/mods/66347/)' ]
+
```

Comment:
Didn't notice it before .. but this addition to the masterlist will not work at all like this.

We are in the `plugins` node of the masterlist, so LOOT expects **plugins** (`.esm`, `.esp` or `.esl` files)
within `-name: ''`

Technically it's doable, but only under `req` or `inc`, so e.g.
```yaml
  - name: 'Example.esp'
    req:
      - name: 'nvse/plugins/Test.dll'
        display: '[Name](Link)'
```

Adding an entry with `-name: 'something.dll'` (as a main `name` tag) will not work, LOOT will simply not show anything in it's UI.

What could be done is to add the following within the `globals` node of the masterlist:
```yaml
# Use instead of NVTF
  - <<: *useInstead
    subs: [ '[NVTF - New Vegas Tick Fix](https://www.nexusmods.com/newvegas/mods/66537/)' ]
    condition: 'file("NVSE/Plugins/sr_New_Vegas_Stutter_Remover.dll")
  - <<: *useInstead
    subs: [ '[New Vegas Heap Replacer](https://www.nexusmods.com/newvegas/mods/69779/)' ]
    condition: 'file("NVSE/Plugins/sr_New_Vegas_Stutter_Remover.dll")
  - <<: *useInstead
    subs: [ '[lStewieAl''s Tweaks and Engine Fixes](https://www.nexusmods.com/newvegas/mods/66347/)' ]
    condition: 'file("NVSE/Plugins/sr_New_Vegas_Stutter_Remover.dll")
```
But even then .. the `useInstead` message reads: `Use {0} instead.`

The 3 different mods are correctly linked like this, but the message wouldn't convey for *what* these mods should be used instead.

Soo .. I'm not sure what to do. Perhaps we need to create a new message anchor for this.

Reply:
It seems like a new message anchor would be in order yes. New Vegas Stutter Remover is mainly used to untie the game's physics to its framerate and reduce microstutter (which unmodded is atrocious). It also is used to improve the stability of the game. NVTF does this already, alongside the two other mods underneat (especially for the stability part, those latter two). 

From what I understand NVTF patches the same methods as NVSR

Reply:
With the following message we could `sub` two mods into the message, which should then work as a `global` message.
```yaml
  - &useXinsteadY
    type: warn
    content:
      - lang: en
        text: 'You are using {0}, but it is recommended to use {1} instead.'
```

Or perhaps:
```yaml
  - &supersede
    type: warn
    content:
      - lang: en
        text: 'You are using {0}, but it is recommended to use {1} as it has superseded {0}.'
```
What's your opinion on this @sibir ?

Reply:
I'm fine with it. Maybe to fit the style of a few messages we already have better, you could use something like `You seem to be using **{0}**, but it has been superseded. It is recommended that you use **{1}** instead.`

Reply:
I'd close the issue and reopen it once the message is added.

Reply:
New message anchor added in https://github.com/loot/prelude/commit/623fd29c315cc5572127f5fb325d3adaad6813a1